Authorization/Authorization/Presentation/Base/FieldsView.swift

```diff
@@ -54,23 +56,67 @@ struct FieldsView: View {
             case .checkbox:
                 EmptyView()
                     .id(index)
-                Text("Checkbox is not support")
             case .plaintext:
-                HTMLFormattedText(
-                    cssInjector.injectCSS(
-                        colorScheme: colorScheme,
-                        html: config.field.label,
-                        type: .discovery,
-                        fontSize: 90, screenWidth: proxy.size.width)
-                )
-                .id(UUID())
-                .padding(.horizontal, -6)
-                
+                plaintext(config: config)
             case .unknown:
                 Text("This field not support")
             }
         }
     }
+
+    @ViewBuilder
+    private func plaintext(config: FieldConfiguration) -> some View {
```

Comment:
the `config` parameter name is conflicting with `self.config`. How about renaming it to `fieldConfig`?